playbooks/roce.yaml

```diff
@@ -0,0 +1,39 @@
+---
+# Playbook for installation SR-IOV DP with CNI and corresponded prerequisites
+- hosts: all
+  become: true
+  become_method: sudo
+  tasks:
+    - include_role:
+        name: roce_backend
+      tags: 
+        - roce_host
+#        - roce_pip
+    
+- hosts: kube-node
+  become: true
+  become_method: sudo
+  tasks:
+    - pip:
+        name: docker-py

```

Comment:
@vtlrazin I had to change this to:
```
    - pip:
        name: docker-py
        state: absent
    - pip:
        name:
        - docker
        - requests>=2.20.1
```

I was getting an error when deploying this roce playbook.
"Failed to import docker or docker-py - No module named ordered_dict."

There's a discussion of this error here: <https://github.com/ansible/ansible/issues/49395>